### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/orb/actions/runs/5270930886#summary-14267025062

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/orb/actions/runs/5270931012#summary-14267025554

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ npm install @memgraph/orb
 ```
 
 Below you can find a simple Typescript example using Orb to visualize a small graph. Feel
-free to check other Javascript examples in `examples/` directory.
+free to check other JavaScript examples in `examples/` directory.
```
Comment (jsoref) [inline README.md:43]: brand

File: src/renderer/canvas/edge/types/edge-curved.ts [hunk 2]
```diff
@@ -55,10 +55,10 @@ export const getCurvedArrowShape = <N extends INodeBase, E extends IEdgeBase>(ed
  *
  * @param {EdgeCurved} edge Edge
  * @param {number} percentage Percentage of the line to get position from
- * @param {IPosition} viaNode Brezier node on the curved line
+ * @param {IPosition} viaNode Bezier node on the curved line
  * @return {IPosition} Position on the line
  */
-const getPointBrezier = <N extends INodeBase, E extends IEdgeBase>(
+const getPointBezier = <N extends INodeBase, E extends IEdgeBase>(
```
Comment (jsoref) [inline src/renderer/canvas/edge/types/edge-curved.ts:61]: I'm hoping I'm right...

File: src/utils/entity.utils.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@
  * A key-value model that keeps an order of the elements by
  * the input sort function.
  *
- * Insipration by NgRx/EntityState: https://github.com/ngrx/platform
+ * Inspiration by NgRx/EntityState: https://github.com/ngrx/platform
```
Comment (jsoref) [inline src/utils/entity.utils.ts:5]: Actually this should be `Inspired by` or `Inspiration from`...

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine

File: src/utils/entity.utils.ts (hunk 3 above)
Comment (tonilastre) [reply src/utils/entity.utils.ts:5]: True, or even `Inspired with`. Feel free to fix it! :)

**Review by tonilastre — APPROVED** [review]
Thanks @jsoref!

Btw we will also add your action as part of the build/verify process. This is really cool stuff!

**Pull request merged**